Packs/CommonScripts/Scripts/CommandLineAnalysis/CommandLineAnalysis.py

```diff
@@ -0,0 +1,669 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import base64
+import re
+import ipaddress
+import json
+
+
+def is_base64(s):
+    """
+    Validates if the provided string is a Base64-encoded string.
+    """
+    try:
+        if isinstance(s, str):
+            s = s.encode('ascii')
+        # Validate Base64
+        padding = len(s) % 4
+        if padding:
+            s += b'=' * (4 - padding)  # Add missing padding
+        base64.b64decode(s, validate=True)  # Use strict validation
+        return True
+    except Exception:
+        return False
+
+
+def clean_non_base64_chars(encoded_str):
+    """
+    Cleans and ensures the Base64 string contains only valid Base64 characters.
+    Adds proper padding if necessary.
+    """
+    # Remove unwanted characters
+    cleaned_str = re.sub(r'[^A-Za-z0-9=]', '', encoded_str)
+    # Fix padding (Base64 strings should be a multiple of 4 in length)
+    padding = len(cleaned_str) % 4
+    if padding:
+        cleaned_str += "=" * (4 - padding)
+    return cleaned_str
+
+
+def remove_null_bytes(decoded_str):
+    """
+    Removes null bytes from the decoded string.
+    """
+    return decoded_str.replace("\x00", "")
+
+
+def decode_base64(encoded_str, max_recursions=5):
+    """
+    Decodes a Base64-encoded string recursively up to a defined limit.
+    """
+    try:
+        recursion_depth = 0
+        while is_base64(encoded_str) and recursion_depth < max_recursions:
+            # Clean and ensure the string is valid
+            encoded_str = clean_non_base64_chars(encoded_str)
+            decoded_bytes = base64.b64decode(encoded_str)
+            try:
+                encoded_str = decoded_bytes.decode('utf-8')
+            except UnicodeDecodeError:
+                encoded_str = decoded_bytes.decode('latin-1')  # Fallback for non-UTF-8 content
+            recursion_depth += 1
+        return encoded_str, recursion_depth > 1  # Return decoded string and if double encoding was detected
+    except Exception as e:
+        print(f"Error decoding base64: {e}")
```

Comment:
Using `print` is not allowed. You can use `CommonServerPython`s `LOG()` function instead.